.goreleaser.yaml

```diff
@@ -17,17 +16,29 @@ builds:
       - linux
       - windows
       - darwin
+
 archives:
   - name_template: "zeabur_{{ .Version }}_{{ .Os }}_{{ .Arch }}"
-    format: binary # don't use archive format, just binary
+    format: binary
+
 checksum:
-  name_template: 'checksums.txt'
+  name_template: "checksums.txt"
+
 changelog:
   use: github
   filters:
     exclude:
-      - '^test:'
-      - '^Merge pull request'
+      - "^test:"
+      - "^Merge pull request"
+
 release:
   extra_files:
     - glob: "README.md"
+
+brews:
+  -
+    repository:
+      owner: zeabur
+      name: homebrew-tap
+    homepage: "https://zeabur.com"
+    description: "Zeabur's official command line tool"
```

Comment:
Maybe we could add a `folder` option? So that we could put other projects into this repo (such as [`zbpack`](https://github.com/zeabur/zbpack))

offical docs: https://goreleaser.com/customization/homebrew/?h=homebrew
```yaml
# Folder inside the repository to put the formula.
folder: Formula
```

How about `folder: zeabur` ?

Reply:
(But I'm not sure if this will affect `brew install zeabur/tap/cli`. (If it only checks for .rb files in the root directory, it may be affected.)

Reply:
We don't need to create a folder, when you want to install `zbpack` it will create another file under `homebrew-tap` called `zbpack.rb`, and you can install it with `brew install zeabur/tap/zbpack` command